wdl/TrainGenotypeFilteringModel.wdl

```diff
@@ -0,0 +1,176 @@
+version 1.0
+
+import "Structs.wdl"
+import "RecalibrateGq.wdl" as recalibrate_gq
+import "TasksMakeCohortVcf.wdl" as tasks_cohort
+
+workflow TrainGenotypeFilteringModel {
```

Comment:
```suggestion
workflow ScoreGenotypes {
```
While there is some "training" functionality built into optimizing cutoffs, the real purpose of this workflow is to assign SL scores